src/api/cli/integration/integration_test.go

```diff
@@ -1341,3 +1357,236 @@ func TestLiveWalletDir(t *testing.T) {
+			func() []string {
+				// Set the third address as change address
+				return []string{"send", "-c", w.Entries[2].Address.String(), w.Entries[0].Address.String(), "1"}
+			},
+			func(t *testing.T, txid string) {},
+		},
+		{
+			"send to one address",
+			func() []string {
+				return []string{"send", w.Entries[0].Address.String(), "2"}
+			},
+			func(t *testing.T, txid string) {},
+		},
+	}
+
+	for _, tc := range tt {
+		t.Run(tc.name, func(t *testing.T) {
+			output, err := exec.Command(binaryPath, tc.args()...).CombinedOutput()
+			if err != nil {
+				fmt.Println("result:", output)
+			}
+			require.NoError(t, err)
+			output = bytes.TrimRight(output, "\n")
+
+			// output: "txid:$txid_string"
+			// split the output to get txid value
+			v := bytes.Split(output, []byte(":"))
+			require.Len(t, v, 2)
+			txid := string(v[1])
+			fmt.Println("txid:", txid)
+			_, err = cipher.SHA256FromHex(txid)
+			require.NoError(t, err)
+
+			// Wait untill transaction is confirmed.
+			for {
+				time.Sleep(time.Second)
+				if isTxConfirmed(t, txid) {
+					break
+				}
+			}
+			tc.checkTx(t, txid)
+		})
+	}
+}
+
+func getTransaction(t *testing.T, txid string) *webrpc.TxnResult {
+	output, err := exec.Command(binaryPath, "transaction", txid).CombinedOutput()
+	require.NoError(t, err)
+
+	var tx webrpc.TxnResult
+	err = json.NewDecoder(bytes.NewReader(output)).Decode(&tx)
+	require.NoError(t, err)
+
+	return &tx
+}
+
+func isTxConfirmed(t *testing.T, txid string) bool {
+	tx := getTransaction(t, txid)
+	return tx.Transaction.Status.Confirmed
+}
+
+// checkCoinhours checks if the address coinhours in transaction are correct
+func checkCoinsAndCoinhours(t *testing.T, tx *webrpc.TxnResult, addr, coins string, coinhours uint64) {
+	addrCoinhoursMap := make(map[string]visor.ReadableTransactionOutput)
+	for _, o := range tx.Transaction.Transaction.Out {
+		addrCoinhoursMap[o.Address] = o
+	}
+
+	o, ok := addrCoinhoursMap[addr]
+	if !ok {
+		t.Fatalf("transaction doesn't have receiver of address: %v", addr)
+	}
+
+	require.Equal(t, coins, o.Coins)
+	require.Equal(t, coinhours, o.Hours)
+}
+
+// prepareAndCheckWallet prepares wallet for live testing.
+// confirms the wallet has at least 2 coins and 2^testCasesNum coinhours.
+func prepareAndCheckWallet(t *testing.T, testCasesNum int) *wallet.Wallet {
+	walletDir, walletName := getWalletPathFromEnv(t)
+	walletPath := filepath.Join(walletDir, walletName)
+	// Checks if the wallet does exist
+	if _, err := os.Stat(walletPath); os.IsNotExist(err) {
+		t.Fatalf("Wallet file: %v does not exist", walletPath)
+	}
+
+	// Loads the wallet
+	w, err := wallet.Load(walletPath)
+	require.NoError(t, err)
+
+	if len(w.Entries) < 3 {
+		// Generates addresses
+		_, err = w.GenerateAddresses(uint64(3 - len(w.Entries)))
+		require.NoError(t, err)
+	}
+
+	coins, hours := getWalletBalance(t, walletPath)
+	if coins < 2 {
+		t.Fatal("Wallet must have at least 2 coins")
+	}
+
+	miniCoinHours := uint64(1 << uint(testCasesNum))
```

Comment:
This depends on `BurnFactor = 2` which will adjust later and is different for other coins